composer.json

```diff
@@ -29,7 +29,7 @@
         }
     ],
     "require": {
-        "php": ">=5.4.0, <=7.1.99"
+        "php": "^5.4 || ^7.0.0 || ^7.1.0 || ^7.2.0"
```

Comment:
To have this consistent, this should be 5.4.0, not 5.4.

Also, for the 7 versions, we need to use ~, not ^, as the ^ would also allow 7.3.0 (if I understand the operator correctly): https://getcomposer.org/doc/articles/versions.md#tilde-version-range-

Reply:
You are correct with the caret operator. Changed accordingly.